includes/CMB2.php

```diff
@@ -410,16 +412,16 @@ public function render_group( $args ) {
 
 		$field_group->peform_param_callback( 'before_group' );
 
-		echo '<div class="cmb-row cmb-repeat-group-wrap ', esc_attr( $field_group->row_classes() ), '" data-fieldtype="group"><div class="cmb-td"><div data-groupid="', esc_attr( $field_group->id() ), '" id="', esc_attr( $field_group->id() ), '_repeat" ', $this->group_wrap_attributes( $field_group ), '>';
+		echo '<div class="cmb-row cmb-repeat-group-wrap ', esc_attr( $field_group->row_classes() ), '" data-fieldtype="group"><div class="cmb-td"><div data-groupid="', esc_attr( $field_group->id() ), '" id="', esc_attr( $field_group->id() ), '_repeat" ', wp_kses( $this->group_wrap_attributes( $field_group ), $this->kses_args( 'group_wrap_attributes' ) ), '>';
```

Comment:
Is it possible to unit test that sort of back-compat stuff? I'd rather have extensive -- even overkill -- unit tests that can tell us verifiably what is likely to break than have to stare at things and hypothesize or run tests manually against really old versions of CMB2 and I'm guessing that would state our case better than "we can't not sanitize this because it could break backwards compatibility" when presenting it to VIP.